website/auth0.erb

```diff
@@ -0,0 +1,59 @@
+<% wrap_layout :inner do %>
```

Comment:
Thanks for also writing a sidebar template, looks quite alright from where I stand. I'll merge this in :)